ui/src/index.html

```diff
@@ -1,139 +1,191 @@
+        <div class="button_wrapper">
+          <button
+            class="button button--display-stats"
+            type="button"
+            id="displayStatsButton"
+          >
+            <span class="button__emoji"> 📊 </span> Display stats
+          </button>
+        </div>
       </div>
-      <span id="managerNotice" class="hidden">
-        The result will be sent to
-        <span id="managerName"></span>
-      </span>
-      <div class="button_wrapper">
-        <button class="button button--display-stats" type="button" id="displayStatsButton">
-          <span class="button__emoji">
-            📊
-          </span> Display stats
-        </button>
+      <div id="displayStats" class="page hidden">
+        <h2>Statistics</h2>
+        <div class="button_wrapper">
+          <button
+            class="button button--display-stats"
+            type="button"
+            id="homeButton1"
+          >
+            <span class="button__emoji"> 🏠 </span> Home
+          </button>
+          <p>Stats by agency:</p>
+          <select id="agencySelector">
+            <!-- TODO: fill this dynamicaly -->
+            <option value="" selected>Global</option>
+            <option value="Paris">Paris</option>
+            <option value="Nantes">Nantes</option>
+            <option value="Rennes">Rennes</option>
+            <option value="Lille">Lille</option>
+            <option value="Lyon">Lyon</option>
+            <option value="Bordeaux">Bordeaux</option>
+            <option value="Grenoble">Grenoble</option>
+            <option value="Montréal">Montréal</option>
+            <option value="Singapour">Singapour</option>
+          </select>
+        </div>
+        <div id="statsTab"></div>
       </div>
-    </div>
-    <div id="displayStats" class="page hidden">
-      <h2>Statistics</h2>
-      <div class="button_wrapper">
-        <button class="button button--display-stats" type="button" id="homeButton1">
-          <span class="button__emoji">
-            🏠
-          </span> Home
-        </button>
+      <div id="recordingPage" class="page hidden">
+        <h2>Sit tight!</h2>
+        <h1>I'm recording your vote...</h1>
+        <div class="loader">
+          <svg class="circular" viewBox="25 25 50 50">
+            <circle
+              class="path"
+              cx="50"
+              cy="50"
+              r="20"
+              fill="none"
+              strokeWidth="2"
+              strokeMiterlimit="10"
+            />
+          </svg>
+        </div>
       </div>
-      <div id="statsTab"></div>
-
-    </div>
-    <div id="recordingPage" class="page hidden">
-      <h2>Sit tight!</h2>
-      <h1>I'm recording your vote...</h1>
-      <div class="loader">
-        <svg class="circular" viewBox="25 25 50 50">
-          <circle class="path" cx="50" cy="50" r="20" fill="none" strokeWidth="2" strokeMiterlimit="10" />
-        </svg>
+      <div id="thankYouPage" class="page hidden">
+        <h2>Thanks!</h2>
```

Comment:
a h2 before h1 ?
Could your remove the h2, and use a

tag with the right CSS styles ?